apps/projects/templates/meinberlin_projects/includes/project_list_tile.html

```diff
@@ -3,8 +3,9 @@
 <div class="tile">
     <div class="tile__head">
         <a tabindex="-1" class="tile__image tile__image--shadowed"
+           {% if project.image %}style="background-image: url({{ project.image|thumbnail_url:'project_thumbnail' }})"{% endif %}
            href="{% extproject_url project 'project-detail' %}" target="{{ project|is_external|yesno:"_blank,_self" }}">
-            <img src="http://placehold.it/285x170" alt="">
+            <span class="sr-only">{{ project.name }}</span>
```

Comment:
I can do that, however W3C says: 
> If an element with a role of presentation is focusable, or otherwise interactive, user agents must ignore the normal effect of the role and expose the element with implicit native semantics, in order to ensure that the element is both understandable and operable.

That basically means the role will get ignored, right?